addons/armaos/functions/fnc_terminal_init.sqf

```diff
@@ -0,0 +1,90 @@
+// Useful when working with custom shortcuts. Contains defines for key codes.
+#include "\a3\ui_f\hpp\definedikcodes.inc"
+
+params ["_computer"];
+
+if (!dialog) then
+{
+	private _ok = createDialog "AE3_ArmaOS_Main_Dialog";
+	if (!_ok) then {hint "Dialog couldn't be opened!"};
+};
+
+private _consoleDialog = findDisplay 15984;	
+private _consoleOutput = _consoleDialog displayCtrl 1100;
+
+/*
```

Comment:
Dead code?

Reply:
removed in commit https://github.com/y0014984/Advanced-Equipment/pull/90/commits/bdeec0439c3a89ef1583c3b663196a37a6c51e0b

Reply:
I'm not sure why this line is here: private _handle = [_computer] spawn AE3_power_fnc_showBatteryLevel;

The 2 lines related to ip address need to be reworked as the whole network stuff in general.